crates/ln-dlc-node/src/ln/coordinator_event_handler.rs

```diff
@@ -381,11 +383,15 @@ where
     // in mind, if the coordinator (the only party that can handle this event) has a channel
     // with the target of this payment we must treat this interceptable HTLC as a regular
     // payment.
+    //
+    // Note, that scenario should not be happening, as we only create an fake scid when creating an
+    // onboarding channel. Any other invoice should create the invoice hint with the actual short
+    // channel id.
```

Comment:
:wrench: I agree with your changes and the comment, but I think this just means we need to remove this lengthy comment.

We can just log a warning if we hit this path instead, as it is unexpected but also not necessarily a problem.